src/main/kotlin/com/ampnet/reportserviceeth/config/ApplicationProperties.kt

```diff
@@ -37,7 +37,7 @@ class ChainProperties {
     var callerAddress: String = "0x0000000000000000000000000000000000000000"
     var cfManagerFactoryAddress: String = ""
     var payoutManagerFactoryAddress: String = ""
-    var startBlockNumber: Long = 18_306_544
+    var startBlockNumber: Long = 18_330_442
```

Comment:
How do you know `startBlockNumber` if you didn't set `cfManagerFactoryAddress`?

Reply:
cfManagerFactoryAddress is overridden in application.properties for mumbai.

Reply:
Ok, then either set both values here on none.